src/odemis/driver/hamamatsurx.py

```diff
@@ -0,0 +1,2043 @@
+        self._metadata[model.MD_SW_VERSION] = self._swVersion
+        self._metadata[model.MD_DET_TYPE] = model.MD_DT_INTEGRATING
+
+        # output CameraInfo should be:
+        # 'Product number: C13440-20C'
+        # 'Serial number: 301730'
+        # 'Firmware: 4.20.B'
+        # 'Version: 4.20.B03-A19-B02-4.02'
+
+        # sensor size (resolution)
+        # Note: sensor size of OrcaFlash is actually much larger (2048px x 2048px)
+        # However, only a smaller subarea is used for operating the streak system.
+        resolution = (int(parent.CamParamGet("Setup", "HWidth")[0]),
+                       int(parent.CamParamGet("Setup", "VWidth")[0]))
+        self._metadata[model.MD_SENSOR_SIZE] = resolution
+
+        # 16-bit
+        self._shape = resolution + (2 ** 16,)
+
+        # physical pixel size is 6.5um x 6.5um
+        sensor_pixelsize = (6.5e-06, 6.5e-06)
+        # self.pixelSize = model.VigilantAttribute(pixelsize, unit="m", readonly=True)
+        self._metadata[model.MD_SENSOR_PIXEL_SIZE] = sensor_pixelsize   #self.pixelSize.value
+
+        self._binning = self._getBinning()  # used by _setResolution
+
+        # need to be before binning, as it is modified when changing binning
+        _resolution = (int(resolution[0]/self._binning[0]), int(resolution[1]/self._binning[1]))
+        self.resolution = model.ResolutionVA(_resolution, ((1, 1), resolution), setter=self._setResolution,
+                                             readonly=True)
+        self._metadata[model.MD_RESOLUTION] = self.resolution.value
+
+        choices_bin = self._getReadoutCamBinningChoices()
+        self.binning = model.VAEnumerated(self._binning, choices_bin, setter=self._setBinning)
+        self._metadata[model.MD_BINNING] = self.binning.value
+
+        # TODO _resolution = physical pixelsize * _binning / _magnification
+        # TODO in odemis resolution is number of pixels, resolution in odemis equals pixelsize
+        # TODO magnification only affects horizontal aka wavelength direction not time axis
+        # TODO does pixelSize include magnification but not binning? should include binning? why extra in updateWavelengthList?
+        magnification = float(1/1.)  # or 1/2.1 get value from streak Lens!!
+        pixelsize = sensor_pixelsize[0] * self._binning[0] / magnification
+        self.pixelSize = model.VigilantAttribute(pixelsize, unit="m", readonly=True)
+        self._metadata[model.MD_PIXEL_SIZE] = self.pixelSize
+
+        # Note: no function to get current acqMode.
+        # Note: Acquisition mode, needs to be before exposureTime!
+        # Acquisition mode should be either "Live" (non-sync acq) or "SingleLive" (sync acq) for now.
+        self.acqMode = "Live"
+
+        # Note: This is the exposure time in live mode, however it is possible to specify multiple ones.
+        # TODO Might be necessary to have multiple VAs in future for using other acq options in RemoteEx.
```

Comment:
No, if we want to support more acq options, we should just write the exposure time to multiple values in RemoteEx, but for the user there should be only a single `.exposureTime`.